app/merginapi.cpp

```diff
@@ -2292,19 +2261,19 @@ ProjectDiff MerginApi::compareProjectFiles( const QList<MerginFile> &oldServerFi
   return diff;
 }
 
-MerginProjectListEntry MerginApi::parseProjectMetadata( const QJsonObject &proj )
+MerginProject MerginApi::parseProjectMetadata( const QJsonObject &proj )
 {
-  MerginProjectListEntry project;
+  MerginProject project;
 
   if ( proj.isEmpty() )
   {
     return project;
   }
+
   if ( proj.contains( QStringLiteral( "error" ) ) )
   {
-    // TODO: handle project error (might be orphaned project)
-
-    proj.value( QStringLiteral( "error" ) ).toInt( 0 ); // error code
+    // handle project error (user might be logged out / do not have write rights / project is on different server / project is orphaned)
+    project.remoteError = proj.value( QStringLiteral( "error" ) ).toInt( 0 ); // error code
```

Comment:
This is printed in case of remoteError https://github.com/lutraconsulting/input/blob/pmr-project-sync/app/qml/components/ProjectDelegateItem.qml#L260